### PR TITLE
BoCA, mp4v2, freac: updates and fix-ups

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update, fixes.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
Draft for now, since the error is not PPC-specific, but extends to all archs when Xcode is 3.2.6. Upstream should add a fix.

**Comment by barracuda156** [comment]
Ready to merge now.

**Comment by barracuda156** [comment]
@herbygillot @mascguy @reneeotten Could we merge this, please?

**Comment by barracuda156** [comment]
In fact give me some time, I rather fix everything up to `freac` and update the latter in the same PR.

**Comment by barracuda156** [comment]
Ok, if CI pass, everything is ready now.

File: audio/BoCA/Portfile [hunk 1]
```diff
@@ -5,11 +5,15 @@ PortGroup           github 1.0
...
-revision            0
-checksums           rmd160  5665880a576d3f90e7754362be6bfd1b414b3c4d \
-                    sha256  9befa4555b5d9a0ecf3b8472f1d8f14c26dee50993d1d66f35a2d0f140a72870 \
-                    size    2634890
+# Use commit for now due to several fixes from upstream, see:
+# https://github.com/enzo1982/BoCA/issues/12
+# https://github.com/enzo1982/BoCA/pull/13
+github.setup        enzo1982 BoCA 4ea79d518b508d43e16a0b86a313e99342efcc71
+version             1.0.7
+revision            1
```
Comment (herbygillot) [inline audio/BoCA/Portfile:13]: If the `version` increases, then `revision` should return to zero.

Comment (barracuda156) [reply audio/BoCA/Portfile:13]: Missed that. Will fix now. Thanks.

Comment (barracuda156) [reply audio/BoCA/Portfile]: @herbygillot CI passed, all good.

**Pull request merged**